DMX.Core.Api.Tests.Unit/Services/Orchestrations/LabOrchestrationServiceTests.Exceptions.RetrieveById.cs

```diff
@@ -0,0 +1,147 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using DMX.Core.Api.Models.Foundations.Labs;
+using DMX.Core.Api.Models.Orchestrations.Labs.Exceptions;
+using FluentAssertions;
+using Moq;
+using Xeptions;
+using Xunit;
+
+namespace DMX.Core.Api.Tests.Unit.Services.Orchestrations
+{
+    public partial class LabOrchestrationServiceTests
+    {
+        [Theory]
+        [MemberData(nameof(LabDependencyValidationExceptions))]
+        public async Task ShouldThrowOrchestrationDependencyValidationExceptionOnRetrieveByIdIfDependencyValidationErrorOccursAndLogItAsync(
```

Comment:
This line is longer than 120 characters. Change it to this:
`ShouldThrowDependencyValidationOnRetrieveByIdIfDependencyValidationErrorOccurrsAndLogItAsync`